images/cluster-nfd-operator.yml

```diff
@@ -31,3 +31,9 @@ update-csv:
   manifests-dir: manifests/
   registry: image-registry.openshift-image-registry.svc:5000
   valid-subscription-label: '["OpenShift Kubernetes Engine", "OpenShift Container Platform", "OpenShift Platform Plus"]'
+  image-references-path: manifests/image-references
+image_has_bundle: true
+image_references:
```

Comment:
Disagree. This can be calculated cheaply and deterministically when initializing the tree. Extra config will open possibilities for mistakes.